src/isc-dhcp/patch/0004-Support-for-loading-port-alias-map-file-to-replace-p.patch

```diff
@@ -72,26 +72,26 @@ index 0f7d658..797dac6 100644
 +				usage(use_noarg, argv[i-1]);
 +			if (load_interface_alias_map(argv[i]) != 0)
 +				log_fatal("Failed to load interface name-alias map.");
- 		} else if (!strcmp(argv[i], "--version")) {
- 			log_info("isc-dhcrelay-%s", PACKAGE_VERSION);
- 			exit(0);
-@@ -726,6 +745,7 @@ main(int argc, char **argv) {
+  		} else if (argv[i][0] == '-') {
+ 			usage("Unknown command: %s", argv[i]);
+  		} else {
+@@ -841,6 +860,7 @@ main(int argc, char **argv) {
  	dispatch();
  
  	/* In fact dispatch() never returns. */
 +	free_interface_alias_map();
  	return (0);
  }
  
-@@ -1151,6 +1171,7 @@ format_relay_agent_rfc3046_msg(const struct interface_info *ip, struct dhcp_pack
+@@ -1271,6 +1291,7 @@ format_relay_agent_rfc3046_msg(const struct interface_info *ip, struct dhcp_pack
  					*/
  					if (packet->htype && !packet->giaddr.s_addr) {
  						int ret = 0, vlanid = 0;
-+						char ifalias[IFNAMSIZ] = { 0 };
+ 						char ifalias[IFNAMSIZ] = { 0 };
```

Comment:
I believe you removed the `+` from the start of the line with your latest commit. This will break the patch. Have you tested it?

Reply:
Updated.